src/Type/ContextDynamicReturnTypeExtension.php

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types = 1);
+
+namespace SaschaEgerer\PhpstanTypo3\Type;
+
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\Reflection\ParametersAcceptorSelector;
+use PHPStan\Type\DynamicMethodReturnTypeExtension;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\Type;
+use TYPO3\CMS\Core\Context\Context;
+use TYPO3\CMS\Core\Context\DateTimeAspect;
+use TYPO3\CMS\Core\Context\LanguageAspect;
+use TYPO3\CMS\Core\Context\TypoScriptAspect;
+use TYPO3\CMS\Core\Context\UserAspect;
+use TYPO3\CMS\Core\Context\VisibilityAspect;
+use TYPO3\CMS\Core\Context\WorkspaceAspect;
```

Comment:
```suggestion
```

Does it help for TYPO3 8.7 if we remove the use statements (add FQN below) and then check in the isMethodSupported if the interface `TYPO3\CMS\Core\Context\AspectInterface` does exist ?

Reply:
Using the ::class constants still broke it.
I now used the static strings, this makes the pipeline pass

Reply:
I can't reproduce this. Using `::class` should work fine if you use the FQN and do not `use` it. 
See https://3v4l.org/hTgfM